pkgs/applications/misc/dwdiff/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchurl
+, gettext
+, pkg-config
+, icu
+}:
+
+stdenv.mkDerivation rec {
+  pname = "dwdiff";
+  version = "2.1.4";
+
+  src = fetchurl {
+    url = "https://os.ghalkes.nl/dist/${pname}-${version}.tar.bz2";
+    sha256 = "sha256-3xb+xE3LRn1lpCRqQ2KPk3QZlsF3PpMLkMbd4i3Vjgo=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [
+    gettext
+    icu
+  ];
+
+  meta = with lib; {
+    description = "An application to split, merge, rotate and mix PDF files";
```

Comment:
maybe `"a word-level diff with color support which lets you specify delimiter- and whitespace-characters"` ?